First_project/First_project/Profile/LoginViewController.swift

```diff
@@ -0,0 +1,218 @@
+//
+//  LoginViewController.swift
+//  First_project
+//
+//  Created by Taras Kyparenko on 03.05.2022.
+//
+
+import UIKit
+
+class LoginViewController: UIViewController {
+    
+    lazy var scrollView: UIScrollView = {
+        let view = UIScrollView()
+        view.backgroundColor = .white
+        view.showsVerticalScrollIndicator = true
+        view.delegate = self
+        return view
+    }()
+    
+    private lazy var areaView: UIView = {
+        let view = UIView()
+        return view
+    }()
+    
+    lazy var logo: UIImageView = {
+        let image = UIImage(named: "LogoVK")
+        var logoPicture = UIImageView(image: image)
+        return logoPicture
+    }()
+    
+    lazy var profileLogin: UITextField = {
+        let login = UITextField()
+        login.setLeftPaddingPoints()
+        login.setRightPaddingPoints()
+        login.font = UIFont.systemFont(ofSize: 16, weight: .regular)
+        login.autocapitalizationType = .none
+        login.returnKeyType = .done
+        login.delegate = self
+        login.layer.cornerRadius = 10
+        login.layer.borderWidth = 0.5
+        login.layer.borderColor = UIColor.lightGray.cgColor
+        login.autocorrectionType = .yes
+        login.textColor = .black
+        login.autocapitalizationType = .none
+        login.backgroundColor = .systemGray6
+        login.placeholder = "Email or phone nomber"
+        return login
+    }()
+    
+    lazy var profilePassword
```

Comment:
Most of the properties are the same with profileLogin.
Consider to extract them to the method to avoid repetition

Reply:
done